### ISSUE TITLE
Attempting to attack yourself while in f6 will crash the game.

### ISSUE THREAD

**Issue by Romoslayer** [Zergatul/freecam#1]
Attempting to attack yourself while in f6 will crash the game, using f6 also doesn't show your skin.

States: Player Romoslayer tried to attack an invalid entity

logs
[latest.log](https://github.com/Zergatul/freecam/files/8721989/latest.log)

Mod version: 1.18.2-0.1 Beta
Forge version: 1.18.2-40.1.0

**Comment by Zergatul** [comment]
Do you have other mods?

**Comment by Romoslayer** [comment]
I only had client side mods installed.
![image](https://user-images.githubusercontent.com/47157784/169886172-c34aacc9-caf5-4449-af4a-97985887fbe0.png)
These are the only mods that I used and the crash still happened.

**Comment by Zergatul** [comment]
Sorry, I don't know what these mods do. What do you mean by "I am also able to attack entities within range of my player while in free cam"? How are you doing this?

**Comment by Zergatul** [comment]
Thanks for the video. I am playing only with optifine, I am unable to attack entities in freecam (or placing blocks). As I understand you get crash with "Attempting to attack yourself" message when you attack yourself accidentally?

**Comment by Romoslayer** [comment]
> Thanks for the video. I am playing only with optifine, I am unable to attack entities in freecam (or placing blocks). As I understand you get crash with "Attempting to attack yourself" message when you attack yourself accidentally?
> 
> Anyway, I already have fix for player skins, and "Attempting to attack yourself" can also be fixed. Give me 1-2 days.
> 
> BTW, some advanced cheats allows you to place blocks while in freecam or open containers. For example, you can be trapped inside bedrock box, and can still open chest outside. My mod doesn't allow to do this. It was surprise for me you are able to attack entities.

Yes, whenever I attack myself (either on accident or on purpose) my game crashes. I have also tested this with optifine and freecam alone and this issue still persists.

**Comment by Zergatul** [comment]
You are right, I tried to attack entities on the long distance.
I added new release with fixes.

**Comment by Zergatul** [comment]
Haha, you are genius! I didn't know I can open chests in freecam. Advanced cheats use another approach to enter freecam mode, that's why I thought this is impossible and didn't even try.

**Comment by Romoslayer** [comment]
:P
